armi/cases/case.py

```diff
@@ -780,6 +780,10 @@ def writeInputs(
         for a parameter sweep or migration) to be written out as input
         for a forthcoming case.
 
+        .. impl:: Write blueprints settings to disk
+            :id: I_ARMI_BP_TO_DB
+            :implements: R_ARMI_BP_TO_DB
+
```

Comment:
I think this should be moved to `saveToSTream()`:

https://github.com/terrapower/armi/blob/main/armi/reactor/blueprints/gridBlueprint.py#L525

And I think the text should read something like:

> .. impl:: Write a blueprint file from a blueprint object.

And we should move the `.. test::` crumb for this, too.